### PR TITLE
Document the repo create command

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,20 @@ Repository access
 
 After this, the service will be available on https://<node.fqdn>/
 
+
+## Create
```
Comment (bowlofeggs) [inline README.md]: Ah, I guess the bots don't like this style. I wasn't familiar with using ```---```'s to do headings in MD - I'll need to look up a way to make subheadings in that style.

Comment (puiterwijk) [reply README.md]: Let's just switch to atx-style all around, i.e. #, ##, ###, etc.
The check just complains that it's inconsistent, so let's go with consistent atx style.

Comment (bowlofeggs) [reply README.md]: Done!

**Review by puiterwijk — APPROVED** [review]
👍

**Pull request merged**